module/zfs/spa.c

```diff
@@ -4550,6 +4761,8 @@ spa_vdev_attach(spa_t *spa, uint64_t guid, nvlist_t *nvroot, int replacing)
 		return (spa_vdev_exit(spa, NULL, txg, ENOTSUP));
 
 	pvd = oldvd->vdev_parent;
+	if (pvd->vdev_ops == &vdev_mirror_ops && vdev_draid_max_rebuild > 0)
```

Comment:
Since rebuild is not specific to draid, it should not have draid in the name for this parameter.